.buildkite/commands/build-ios.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -eu
 
 echo '--- :node: Setup Node depenendencies'
-npm ci
+npm ci --unsafe-perm --prefer-offline --no-audit --no-progress
```

Comment:
What is the intent with the addition of `--unsafe-perm`?

Reply:
It was intentional in that other commands running in Docker use it, so I ported it for consistency:

https://github.com/wordpress-mobile/gutenberg-mobile/blob/81c9ed87db77693ab8e7c030906850f925698232/.buildkite/pipeline.yml#L47

https://github.com/wordpress-mobile/gutenberg-mobile/blob/81c9ed87db77693ab8e7c030906850f925698232/.buildkite/pipeline.yml#L73

As of whether it's appropriate, I don't know. I didn't dig into it at the time...

From [the docs](https://docs.npmjs.com/cli/v6/using-npm/config#unsafe-perm):

> **unsafe-perm**
>
> – Default: false if running as root, true otherwise
> - Type: Boolean
>
> Set to true to suppress the UID/GID switching when running package scripts. If set explicitly to false, then installing as a non-root user will fail.

If I'm reading it correctly, it seems like it should be redundant given that our scripts don't run as root and therefore the value is true by default.